flask/test/data/base.py

```diff
@@ -0,0 +1,26 @@
+import pytest
+from boxwise_flask.models.base import Base
+
+
+@pytest.fixture()
+def default_base(default_organisation):
+    mock_base = {"id": 1, "seq": 1, "organisation_id": default_organisation["id"]}
+    Base.create(**mock_base)
+    return mock_base
+
+
+@pytest.fixture()
+def default_bases(default_organisation):
+    bases_dict = {}
+    mock_base = {"id": 1, "seq": 1, "organisation_id": default_organisation["id"]}
+    bases_dict[mock_base["id"]] = mock_base
+    Base.create(**mock_base)
+
+    mock_base = {"id": 2, "seq": 1, "organisation_id": default_organisation["id"]}
+    bases_dict[mock_base["id"]] = mock_base
+    Base.create(**mock_base)
+
+    mock_base = {"id": 3, "seq": 1, "organisation_id": default_organisation["id"]}
+    bases_dict[mock_base["id"]] = mock_base
+    Base.create(**mock_base)
```

Comment:
Can we use a loop instead? And write `bases` instead of `bases_dict` (I don't find it good style to include types in variable names)?
```
for i in range(1,4):
    mock_base = {"id": i, ...}
    bases[i] = mock_base
    Base.create(**mock_base)
```

Reply:
I'd agree except when converting something of the same structure to a different type. makes it easier to understand the difference in two things that represent the same thing as seperate types.